docs/get-started.asciidoc

```diff
@@ -0,0 +1,299 @@
+
+With these limited changes to the `Program.cs` file, the application is now configured to use the
+OpenTelemetry SDK and the distro to collect traces and metrics, which are exported via the
+OpenTelemetry protocol (OTLP).
+
+[discrete]
+[[get-started-sample-endpoint]]
+==== Add a sample endpoint
+
+To demonstrate the tracing capabilities, add a simple endpoint to the application:
+
+[source,csharp]
+----
+app.MapGet("/", async (IHttpClientFactory httpClientFactory) =>
+{
+	using var client = httpClientFactory.CreateClient();
+
+	await Task.Delay(100);
+	var response = await client.GetAsync("http://elastic.co"); <1>
+	await Task.Delay(50);
+
+	return response.StatusCode == System.Net.HttpStatusCode.OK ? Results.Ok() : Results.StatusCode(500);
+});
+----
+<1> Using this URL will require two redirects, allowing us to see multiple spans in the trace.
+
+The distro will automatically enable the exporting of signals via the OTLP exporter. This
+exporter requires that endpoint(s) are configured. A common mechanism for configuring
+endpoints is via environment variables.
+
+////
+What does the _minimal_ configuration / setup look like?
+Are there any configuration options that are _required_?
+////
+[discrete]
+[[get-started-configure]]
+==== Configure the distro
+
+To configure the distro, at a minimum you'll need the deployment's OTLP endpoint and
+authorization data to set the appropriate `OTLP_*` environment variables:
+
+* `OTEL_EXPORTER_OTLP_ENDPOINT`: The full URL of the endpoint where data will be sent.
+* `OTEL_EXPORTER_OTLP_HEADERS`: A comma-separated list of `key=value` pairs that will
+be added to the headers of every request. This is typically this is used for
+authentication information.
+
+[NOTE]
+====
+The steps below assume you are using an Elastic Cloud deployment as the destination
+for your observability data.
+====
+
+////
+Step-by-step instructions
+////
+When sending data to Elastic, you can find the values of these variables in Kibana's APM setup guide:
+
+. In {kib}, go to *Setup guides*.
+. Select *Observability*.
+. Select *Monitor my application performance*.
+. Scroll down and select the *OpenTelemetry* option.
++
+image:images/elastic-cloud-opentelemetry-configuration.png[Elastic Cloud OpenTelemetry configuration]
+
+. Configure environment variables for the application either in `launchSettings.json` or in the environment
+where the application is running.
+. Once configured, run the application and make a request to the root endpoint. A trace will be generated
+and exported to the OTLP endpoint.
+
+For more information on all the available configuration options, refer to <<configure>>.
+
+////
+Does the user need to do something after configuring the distro
+for data to start being sent Elastic?
+If no, delete this section.
+////
+[discrete]
+[[get-started-initialization]]
+==== Initialize the distro
+
+////
+Step-by-step instructions
+////
+
+////
+What should they expect to see after running this?
```

Comment:
Not sure I understand, the next section has a successful trace in the waterfall view in the Elastic APM UI. This should suffice?